contrib/tracing/mempool_monitor.py

```diff
@@ -0,0 +1,361 @@
+#!/usr/bin/env python3
+# Copyright (c) 2022 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+""" Example logging Bitcoin Core mempool events using the mempool:added,
+    mempool:removed, mempool:replaced, and mempool:rejected tracepoints. """
+
+import curses
+import sys
+from datetime import datetime, timezone
+
+from bcc import BPF, USDT
```

Comment:
```
test/functional/interface_usdt_mempool.py:14: error: Cannot find implementation or library stub for module named "bcc"  [import]
test/functional/interface_usdt_mempool.py:14: note: See https://mypy.readthedocs.io/en/stable/running_mypy.html#missing-imports
Found 1 error in 1 file (checked 256 source files)
^---- failure generated from lint-python.py
```


I think you may want to do this?
```suggestion
# Test will be skipped if we don't have bcc installed
try:
    from bcc import BPF, USDT # type: ignore[import]
except ImportError:
    pass
```

Reply:
yes, that's the trick we're using in the other tracepoint tests too

Reply:
Thanks.